src/Controller/Admin/Improve/Design/LinkBlockController.php

```diff
@@ -201,27 +195,14 @@ public function updatePositionsAction(Request $request, $hookId)
             'parentId' => $hookId,
         ];
 
-        /** @var PositionDefinition $positionDefinition */
-        $positionDefinition = $this->get('prestashop.module.link_block.grid.position_definition');
```

Comment:
FYI: I introduced `MultiStorePositionUpdater` here: https://github.com/PrestaShop/PrestaShop/pull/21352/files

even with respect for `parentId` etc. but it would be easier to just `updatePositions` using repo in this case

tho I still think that we should have such a tools directly in Core